src/sync/incomingSyncUtils.js

```diff
@@ -306,9 +306,32 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
   let internalRecord;
   switch (recordType) {
     case 'IndicatorAttribute': {
+      internalRecord = {
+        id: record.ID,
+        indicator: database.getOrCreate('ProgramIndicator', record.indicator_ID),
+        description: record.description,
+        code: record.code,
+        index: parseNumber(record.index),
+        isRequired: parseBoolean(record.is_required),
+        valueType: record.data_type && record.data_type.value,
```

Comment:
think you can `record?.data_type?.value`

Reply:
Nice. f0e6d5088ba704dc6ec77e6c22beded334b6890e.